source/Cosmos.System2/Graphics/Fonts/Font.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Cosmos.System.Graphics.Fonts
+{
+    /// <summary>
+    /// Base class for fonts.
+    /// </summary>
+    public abstract class Font
```

Comment:
I thinks its better if a font has a method to get the data for a single letter, because fonts might not have all letters or store all letters in a single array.

Also design and speed wise, I think it would be better if we dont overwrite the parameters Data, Height and Width and rather have a ctor where theses values are set.

Reply:
For now PSF only use an array for the whole character set. If we support more fonts it will be changed after?

Reply:
Yeah that works